extensions/dicom-microscopy/src/DicomMicroscopyANNSopClassHandler.js

```diff
@@ -0,0 +1,103 @@
+import OHIF, { DicomMetadataStore } from '@ohif/core';
+import loadAnnotation from './utils/loadAnnotation';
+import getSourceDisplaySet from './utils/getSourceDisplaySet';
+
+const { utils } = OHIF;
+
+const SOP_CLASS_UIDS = {
+  MICROSCOPY_BULK_SIMPLE_ANNOTATION: '1.2.840.10008.5.1.4.1.1.91.1',
+};
+
+const SOPClassHandlerId =
+  '@ohif/extension-dicom-microscopy.sopClassHandlerModule.DicomMicroscopyANNSopClassHandler';
+
+function _getDisplaySetsFromSeries(instances, servicesManager, extensionManager) {
+  // If the series has no instances, stop here
+  if (!instances || !instances.length) {
+    throw new Error('No instances were provided');
+  }
+
+  const { displaySetService, microscopyService } = servicesManager.services;
+
+  const instance = instances[0];
```

Comment:
Please choose the last instance in the series, after sorting by increasing instance date/time.  That way it becomes possible to "replace" the previous instance, which is otherwise not possible in DICOM.

Reply:
Sort by date/time of the instance, or by increase instance number, or something that should get the most recent one last, and use that one.

Reply:
Updated.